tests/project/project_1/project.yml

```diff
@@ -0,0 +1,20 @@
+
+native_app:
```

Comment:
Few things to thinks about:
1. single vs many apps - if my project contains two apps should I have to ymls or one with two definitions?
2. native_app vs app - should we consider structure like this:
```yml
app:
  type: native_app
  ...
```

This brings all app-things like Streamlit, Native Apps and any future objects under one umbrella of "apps". Implementation-wise this shouldn't be any harder (schema of type one-of). Plus, if we decide to support "one app" per project then it makes the config more explicite. WDYT?

Reply:
IMO for Point 2 - @sfc-gh-turbaszek  what do you envision for bringing more products under the umbrella that are non-app (outside streamlit, native apps) such as SPCS? They would not fit under `app:type:`. And if we define a unified yml file for all these, would `spcs:` belong at the same level as `app:`?

Reply:
In light of today's discussion I want to propose this way forward:
- `native_app:` stays as-is
- the value of "type" is used as the top-level keys, e.g. `streamlit:`

For multiple applications, they should live in different directories of the same repo, with their own `snowflake.yml` files. Use cases like that of the connectors team can be supported by using different overrides files; because they can change any aspect of the project config (as the schema is identical), apps that share most things can be deployed that way, e.g. `cd connectors/ && snow app run -e servicenow.yml`